src/main/webapp/js/controllers/techDetailsController.js

```diff
@@ -324,6 +324,7 @@ angular.module('techGallery').controller(
     				callBackLoaded();
     				$scope.comment = '';
     			});
+    			ga('send', 'event', 'button', 'click', 'comments_add');
```

Comment:
@jfmedeiros1820 Will event category be 'button'? 'And action 'click'?
I suggest the category be 'TechGalleryEvents', action be 'comment' and the value be either 'add' or the technology name.